### PR TITLE
Modify rule S927: LaYC format

### PR DESCRIPTION
**Pull request by sebastien-marichal**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone

### CODE REVIEW

File: rules/S927/csharp/metadata.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
```
Comment (cristian-ambrosini-sonarsource) [inline rules/S927/csharp/metadata.json:1]: Are there any open tickets in order to implement this rule for:
- Cfamily
- Java
- Javascript
? If not you can remove all of them

File: rules/S927/csharp/rule.adoc [hunk 2]
```diff
@@ -107,7 +81,7 @@ Could you run through it one more time? There were minor modifications in the sa
 \[~tamas.vajk] I'm not sure why you swapped the parameter order. I feel like reversing the names is a better illustration of the confusion that could result in param names that don't match...?
 
 === on 22 May 2015, 12:48:42 Tamas Vajk wrote:
-\[~ann.campbell.2] I changed it because ``++yk++`` was not used in the method body, so it already just looked like a typo.
+\[~ann.campbell.2] I changed it because `yk` was not used in the method body, so it already just looked like a typo.
 
 
 I've changed the sample code.
```
Comment (cristian-ambrosini-sonarsource) [inline rules/S927/csharp/rule.adoc:87]: You can create a common `rspecator.adoc` file for both vbnet and csharp

File: rules/S927/vbnet/rule.adoc [hunk 3]
```diff
@@ -1,69 +1,46 @@
...
-
-    Public Sub Add(Difference As Integer) Implements IContract.Add  'Noncompliant, parameter name should be "Count"
-        ' ...
-    End Sub
-
 End Class
 ----
 
-=== Compliant solution
+To avoid any ambiguity in the code, parameters' names should match the initial declaration, whether its initial declaration is from an interface or a base class.
```
Comment (cristian-ambrosini-sonarsource) [inline rules/S927/vbnet/rule.adoc:22]: Educational: aren't partial methods a thing also in vbnet?

**Review by cristian-ambrosini-sonarsource — CHANGES_REQUESTED** [review]
Good! I left a couple of comments

File: rules/S927/vbnet/rule.adoc (hunk 3 above)
Comment (sebastien-marichal) [reply rules/S927/vbnet/rule.adoc:22]: I missed that will update it accordingly, thanks!

**Review by cristian-ambrosini-sonarsource — CHANGES_REQUESTED** [review]
Last round

**Pull request merged**